logstash-core/lib/logstash/util.rb

```diff
@@ -58,7 +48,7 @@ def self.thread_info(thread)
                  end
 
     {
-      "thread_id" => get_thread_id(thread),
+      "thread_id" => get_thread_id(thread), # might be nil for dead threads
```

Comment:
As we are using `get_thread_id` here but the function is removed above, can you please point me to the new/alternative definition of the function or am I missing something ?